src/style/app.module.css

```diff
@@ -3481,3 +3481,153 @@ button[data-testid='createPostBtn'] {
     opacity: 1;
   }
 }
+
+/* Events.tsx */
+
+.borderNone {
+  border: none;
+}
+
+.colorWhite {
+  color: white;
+}
+
+.backgroundWhite {
+  background-color: white;
+}
+
+.maxWidth {
+  max-width: 800px;
+}
+
+.mainContainer {
+  margin-top: 2rem;
+  width: 50%;
+  flex-grow: 3;
+  max-height: 100%;
+  overflow: auto;
+}
+
+.selectType {
+  border-radius: 10px;
+}
+.dropdown__item {
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+.gap {
+  gap: 20px;
+}
+
+.paddingY {
+  padding: 30px 0px;
+}
+
+.eventActionsContainer {
+  display: flex;
+  flex-direction: row;
+  gap: 15px;
+}
+
+.datePicker {
+  border-radius: 10px;
+  height: 40px;
+  text-align: center;
+  background-color: #f2f2f2;
+  border: none;
+  width: 100%;
+}
+
+.modalBodyEvents {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.switchContainer {
+  display: flex;
+  align-items: center;
+}
+
+.switches {
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  flex-wrap: wrap;
+  margin-top: 20px;
+}
+.titlemodalEvents {
+  color: #707070;
```

Comment:
Convert to a variable. There should be some preexisting ones, if not please create it.